src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea: () => number;
+}
 
+function error(): never {
```

Comment:
```suggestion
function showError(): void {
```
Get used to name functions according to these rules
![image](https://user-images.githubusercontent.com/96409563/227862960-d2ee2674-3e5e-4637-920a-308a8673c636.png)